src/Utils/get.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Type-safe access of deep property of an object
+ *
+ * Thanks! https://github.com/claudepache/es-optional-chaining/issues/2#issuecomment-318680932
+ *
+ * @param obj                     Object to get deep property
+ * @param unsafeDataOperation     Function that returns the deep property
+ * @param valueIfFailOrUndefined  Value to return in case if there is no such property
+ */
+export function get<O, T>(
+  obj: O,
+  unsafeDataOperation: (x: O) => T,
+  valueIfFailOrUndefined?: any
```

Comment:
Shouldn’t this be `T` instead of `any`?

Reply:
It could be `T`... I suppose that would make sense. Then it would enforce that your fallback case matches what you expect the function to give...

Yeah, I'm 👍

Reply:
Will play with it a bit, this was mostly copied over from the example (linked at the top) 👍